config/config.yml

```diff
@@ -54,3 +55,8 @@ networks:
 retry:
     attempts: 20
     linear_delay: 1s
+prometheus:
```

Comment:
Prometheus shouldn't be handled separately and in this case locally, it should be deployed within the environment. and be specific to that environment. It'll differ between compose/k8s.

Reply:
If this is resolved, I'd expect this config to be removed.